File: FlightAppAngular/src/app/model/schedule.ts
import { Time } from "@angular/common";
import { ScheduleDaysService } from "app/service/schedule-days.service";
import { ScheduleDays } from "./scheduleDays";

export class Schedule {
    id: number;
    flightId: number;
    source: string;
    destination: string;
    takeOffTime: Time;
    landingTime: Time;
    scheduleDays: string;
    schedule: Array<ScheduleDays>;

    constructor(private getScheduleDays: ScheduleDaysService) {
        this.id = 0;
        this.source = "";
        this.destination = "";
        this.flightId = 0;
        this.scheduleDays = "";
        this.schedule = this.getScheduleDays.getScheduleDays();
    }
}

File: AirlineMicroService/Model/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineMicroService.Model
{
    public class Schedule
    {
        public int Id { get; set; }
        public int FlightId { get; set; }
        public string Source { get; set; }
        public string Destination { get; set; }
        public TimeSpan TakeOffTime { get; set; }
        public TimeSpan LandingTime { get; set; }
        public string ScheduleDays { get; set; }
        public Flight Flight { get; set; }
    }
}
